books/free-programming-books-subjects.md

```diff
@@ -255,6 +255,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Readings in Database Systems, 5th Ed.](http://www.redbook.io)
 * [Temporal Database Management](http://people.cs.aau.dk/~csj/Thesis/) - Christian S. Jensen
 * [The Theory of Relational Databases](http://web.cecs.pdx.edu/~maier/TheoryBook/TRD.html)
+* [Database Management Systems/Managing Database](https://ebooks.lpude.in/management/mba/term_3/DCAP204_MANAGING_DATABASE_DCAP402_DATABASE_MANAGEMENT_SYSTEMS.pdf) - Dr Anil Sharma (PDF)
```

Comment:
@YashSalunke12 is the book free to be used by people outside LPU?

Reply:
@shrn01 bro can you tell me, what should I change in this? sorry but this is my first time contributing an project! why build failed?

Reply:
Hey don't worry. Let me help you out.